Asterisk.2013/Asterisk.NET/Manager/Originate.cs

```diff
@@ -141,22 +141,6 @@ public long Timeout
 
         #endregion
 
-        #region Variable
-
-        /// <summary>
-        ///     Get/Set the variables to set on the originated call.<br />
-        ///     Variable assignments are of the form "VARNAME=VALUE". You can specify
-        ///     multiple variable assignments separated by the '|' character.<br />
-        ///     Example: "VAR1=abc|VAR2=def" sets the channel variables VAR1 to "abc" and VAR2 to "def".
-        /// </summary>
-        public string Variable

```

Comment:
Mark is obsolete then with compiler error set to true. That way, if someone is using it in their own code, at least it's not just vanished! The error in the attribute should point them in the right direction.